assignment1/remove_duplicates.rb

```diff
@@ -0,0 +1,13 @@
+def non_duplicated_values dupl_arr
+    uniq_arr = [];
+    dupl_arr.each { |val|
+        if !uniq_arr.include? val
+            uniq_arr = uniq_arr.push(val)
+        else
+            uniq_arr.delete(val)
```

Comment:
why deletion is needed here?

Reply:
So It value that is repeated don't come in o/p

Reply:
why it wil come? you are using a new array